.circleci/config.yml

```diff
@@ -374,16 +370,16 @@ workflows:
         - equal: [ true, << pipeline.parameters.upkeep >> ]
     jobs:
       - upkeep
-  # link-validation-workflow:
-  #   triggers:
-  #     - schedule:
-  #         cron: '0 10 * * 1,4'
-  #         filters:
-  #           branches:
-  #             only:
-  #               - stage
-  #   jobs:
-  #     - link-validation
+  link-validation-workflow:
+    triggers:
+      - schedule:
+          cron: '0 10 * * 1,4'
```

Comment:
We should set up a trigger to handle the runs for this if it should be ran regularly like upkeep and scheduled backup.

Reply:
i have it set on a cron to run on monday and thursday. or are you saying this is not the correct way to set it up?